es-runtime/src/web-components/PanelHeaderBar.ts

```diff
@@ -205,4 +205,6 @@ export class PanelHeaderBar extends HTMLElement {
         })
     }
 }
-window.customElements.define('es-panel-header-bar', PanelHeaderBar)
+if(!window.customElements.get('es-panel-header-bar')) {
+    window.customElements.define('es-panel-header-bar', PanelHeaderBar)
```

Comment:
this prevents a console error complaining that 'es-panel-header-bar' has already been declared